server-web/actix-rest/src/main.rs

```diff
@@ -21,16 +21,19 @@ pub async fn fetch_flights() -> HttpResponse {
 }
 
 /// Responds a POST request with a boolean value.
-pub async fn request_flights() -> HttpResponse {
+///
+/// Expects a JSON body conforming to the [`FlightInput`] struct.
+pub async fn request_flights(flight: web::Json<FlightInput>) -> HttpResponse {
+    print!("{:?}", flight);
```

Comment:
This might affect the benchmark, we'll review this sort of thing later. We'll do a PR which removes extra stuff across all stacks to try to get the benchmark tasks as similar as possible